app_config_b2c.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+b2c_tenant = "fabrikamb2c"
+signupsignin_user_flow = "b2c_1_signupsignin1"
+editprofile_user_flow = "b2c_1_profileediting1"
+authority_template = "https://{tenant}.b2clogin.com/{tenant}.onmicrosoftonline.com/{user_flow}"
+
+CLIENT_SECRET = "Enter_the_Client_Secret_Here" # Our Quickstart uses this placeholder
+# In your production app, we recommend you to use other ways to store your secret,
+# such as KeyVault, or environment variable as described in Flask's documentation here
+# https://flask.palletsprojects.com/en/1.1.x/config/#configuring-from-environment-variables
+# CLIENT_SECRET = os.getenv("CLIENT_SECRET")
+# if not CLIENT_SECRET:
+#     raise ValueError("Need to define CLIENT_SECRET environment variable")
+
+AUTHORITY = authority_template.format(
+    tenant=b2c_tenant, user_flow=signupsignin_user_flow)
+PROFILE_AUTHORITY = authority_template.format(
+    tenant=b2c_tenant, user_flow=editprofile_user_flow)
+
+CLIENT_ID = "Enter_the_Application_Id_here"
+
+REDIRECT_PATH = "/getAToken"  # It will be used to form an absolute URL
+    # And that absolute URL must match your app's redirect_uri set in AAD
+
+# This is the resource that you are going to access in your B2C tenant
+ENDPOINT = ''
+
+# These are the scopes that you defined for the web API
+SCOPE = ["demo.read", "demo.write"]
```

Comment:
>SCOPE [](start = 0, length = 5)

the other samples are targeting a node js web api, which validates the token using the passport lib. not sure how you guys handle that stuff in python. but the scopes we use are: `public static string[] ApiScopes = { "[https://fabrikamb2c.onmicrosoft.com/helloapi/demo.read](https://fabrikamb2c.onmicrosoft.com/helloapi/demo.read)" };` and the api endpoint is: `public static string ApiEndpoint = "[https://fabrikamb2chello.azurewebsites.net/hello](https://fabrikamb2chello.azurewebsites.net/hello)";`

Reply:
@rayluo @abhidnya13 : here are apps coordinates you might want to have a look at,
https://github.com/Azure-Samples/active-directory-b2c-dotnet-desktop/blob/365e548fe2ef9954188b9332d62c82ce3e56904d/active-directory-b2c-wpf/App.xaml.cs#L13-L25

Reply:
Let's bring this to [this discussion](https://github.com/Azure-Samples/ms-identity-python-webapp/pull/4#discussion_r341660716) that @jennyf19 and @mmacy are planning to have. It is more about what default values a sample should have: something we own, or something generic for customer.